app/src/main/java/com/poupa/vinylmusicplayer/dialogs/BottomSheetDialog/BottomSheetDialogWithButtons.java

```diff
@@ -53,6 +53,7 @@ public BottomSheetDialogWithButtons setDefaultButtonIndex(int defaultIndex) {
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.bottom_sheet_button_list, container, false);
         Context context = getContext();
+        expand();
 
```

Comment:
Suggestion: is it possible to defer this call to later stage, where the bottom sheet is available with certainty? This would allow us to avoid having the runWhenBottomSheetAvailable entirely.

Reply:
I could add a list containing code that gets ran onCreate, would that work for you?
Then the problem will be shifted to the memory side instead the cpu side.